api/elastic.py

```diff
@@ -389,6 +393,24 @@ def createFilterClausesAndAggregations(self, filterParams):
                 'filter', **{'terms': {'editions.formats': formats}}
             )
 
+        if len(govDocFilters) > 0:
+            # onlyGovDoc|noGovDoc|all
+            # if govDocFilter is all: skip
+            # elif govDocFilter is no: is_government_document: false
+            # elif govdocFilter is only: is_gov_doc: true
+            if govDocFilters[0][1] == 'noGovDoc':
+                self.govDocFilters = [
+                    Q('term', **{'is_government_document': False})
+                ]
```

Comment:
I don't think this needs to be an array, because it's a boolean filter it only makes sense for it to only be set once. So this could be `self.govDocFilter`, which should simplify things below!